doc/user_manual_en.md

```diff
@@ -240,6 +241,10 @@ Additionally, qTox supports three modes of Markdown parsing:
 
 *Note that any change in Markdown preference will require a restart.*
 
+qTox also supports action messages by prefixing a message with `/me`, where
+`/me` is replaced with your current username. For example `/me likes cats`
+turns into *`` * qTox User``*` likes cats`.
```

Comment:
uh, sorry, what I wrote was kind misleading – it was supposed to mean that whole sentence, not just user name turns italic